### PR TITLE
Blog on IP by abhinav

### PR DESCRIPTION
**Pull request by rana-abhishek**

### CODE REVIEW

File: source/_posts/potential-patent-infringement.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/potential-patent-infringement.md:1]: Link to   previous content is pending

File: source/_posts/potential-patent-infringement.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+title: How to check whether your product causes potential patent infringement
+date: 2020-07-24 18:16:09
+author: Abhinav Arora
+category:
+  - ["IP-Patents"]
+tags:
```
Comment (Rajeev111974) [inline source/_posts/potential-patent-infringement.md:7]: Keyword   should be linked with website,

File: source/_posts/potential-patent-infringement.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - Patents
+  - Infringement
+---
+
```
Comment (Rajeev111974) [inline source/_posts/potential-patent-infringement.md:11]: Content   should have link to previous content
--

File: source/_posts/potential-patent-infringement.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: How to check whether your product causes potential patent infringement
```
Comment (Rajeev111974) [inline source/_posts/potential-patent-infringement.md:2]: Image should be optimized with Alt text

File: source/_posts/potential-patent-infringement.md (hunk 2 above)
Comment (Rajeev111974) [inline source/_posts/potential-patent-infringement.md:7]: 1-2 long tail keyword should be there if possible

**Comment by Rajeev111974** [comment]
HI Abhishek,

 

Please check blog we have put our feedback there.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/potential-patent-infringement.md (hunk 1 above)
Comment (rana-abhishek) [reply source/_posts/potential-patent-infringement.md:1]: I am creating this blog for Abhinav and this is his first blog on our website so currently we dont have any links. I have added couple of links of his previous post in between content as per given in word doc by him.

File: source/_posts/potential-patent-infringement.md (hunk 2 above)
Comment (rana-abhishek) [reply source/_posts/potential-patent-infringement.md:7]: Added two keywords

File: source/_posts/potential-patent-infringement.md (hunk 3 above)
Comment (rana-abhishek) [reply source/_posts/potential-patent-infringement.md:11]: I am creating this blog for Abhinav and this is his first blog on our website so currently we dont have any links. I have added couple of links of his previous post in between content as per given in word doc by him.

File: source/_posts/potential-patent-infringement.md (hunk 4 above)
Comment (rana-abhishek) [reply source/_posts/potential-patent-infringement.md:2]: We already have alt text added to image.

File: source/_posts/potential-patent-infringement.md (hunk 2 above)
Comment (rana-abhishek) [reply source/_posts/potential-patent-infringement.md:7]: We have added two `Nayan IP, Nayan data protection`

File: source/_posts/potential-patent-infringement.md (hunk 3 above)
Comment (Rajeev111974) [reply source/_posts/potential-patent-infringement.md:13]: WAITING FOR NAYAN LINK

**Comment by Rajeev111974** [comment]
HI Abhishek,

 

Blog reviewed and ready for live, please do the thing needful.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

**Pull request merged**